man/systemd.network.xml

```diff
@@ -1386,6 +1386,15 @@
             read via <function>sd_network_link_get_sip_servers()</function> function.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>UseSMTPServer=</varname></term>
+          <listitem>
+            <para>When true (the default), the SMTP servers received
+            from the DHCP server will be saved at the state files and can be
+            read via <function>sd_network_link_get_SMTP_servers()</function> function.</para>
```

Comment:
The same comment as I made for POP3 server in the other PR: I don't see much reason to have this option. Let's just always export the data since we don't use it internally.